service/src/main/java/bio/terra/workspace/app/controller/shared/ControllerUtils.java

```diff
@@ -0,0 +1,47 @@
+package bio.terra.workspace.app.controller.shared;
+
+import bio.terra.workspace.generated.model.ApiJobReport;
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.http.HttpStatus;
+
+public class ControllerUtils {
+
+  /**
+   * Returns the result endpoint corresponding to an async request, prefixed with a / character. The
```

Comment:
The first sentence here doesn't seem right. It doesn't seem to be prefixed with a / , maybe {servletpath}/